### PR TITLE
Review howtos

### PR DESCRIPTION
**Pull request by rkratky**
* Heading markup made consistent with starter pack
* Proper inline markup used; semantic where possible
* Language scrubbed

(Fixes [FR-5768](https://warthogs.atlassian.net/browse/FR-5768))

[FR-5768]: https://warthogs.atlassian.net/browse/FR-5768?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: doc/howto/autoinstall-quickstart-s390x.rst [hunk 1]
```diff
@@ -1,33 +1,31 @@
...
+.. _autoinstall-quick-start-s390x:
 
-Automatic installation quick start for s390x
-********************************************
+Autoinstall quick start for s390x
+=================================
 
-This how-to provides basic instructions to perform an automatic installation
-in a virtual machine (VM) on a local machine on the s390x architecture.
+This guide provides instructions to use autoinstall with a current version of Ubuntu for the s390x architecture in a virtual machine (VM) on your computer.
```
Comment (dbungert) [inline doc/howto/autoinstall-quickstart-s390x.rst:6]:
```suggestion
This guide provides instructions on how to use autoinstall with a current version of Ubuntu for the s390x architecture in a virtual machine (VM) on your computer.
```
or similar

File: doc/howto/autoinstall-quickstart-s390x.rst [hunk 2]
```diff
@@ -97,26 +101,29 @@ Add the default user to the ``kvm`` group:
...
         -kernel ~/iso/boot/kernel.ubuntu \
         -initrd ~/iso/boot/initrd.ubuntu \
         -append 'autoinstall ds=nocloud-net;s=http://_gateway:3003/ console=ttysclp0'
 
-The above commands boot the virtual machine, download the configuration from the server
-(prepared in the previous step) and run the installation.
-
-The installer reboots at the end. The ``-no-reboot`` flag to ``kvm`` instructs ``kvm``
-to terminate on reboot. The procedure takes approximately 5 minutes.
+This command boots the VM, downloads the configuration from the server (prepared in the previous step) and runs the installation. The installer reboots at the end. The ``-no-reboot`` option to the ``kvm`` command instructs ``kvm`` to exit on reboot. This command takes approximately 5 minutes.
```
Comment (dbungert) [inline doc/howto/autoinstall-quickstart-s390x.rst:115]: I wonder if we should remove the reference to "takes approximately 5 minutes" outright.  Install times can vary wildly depending on hardware and network and what version is being installed.

File: doc/howto/autoinstall-quickstart.rst [hunk 3]
```diff
@@ -1,48 +1,39 @@
-.. _autoinstall_quickstart:
+.. _autoinstall_quick-start:
 
 Autoinstall quick start
-***********************
+=======================
 
-The intent of this page is to provide simple instructions to perform an
-autoinstall in a VM on your machine.
+This guide provides instructions to use autoinstall with a current version of Ubuntu for the amd64 architecture in a virtual machine (VM) on your computer.
```
Comment (dbungert) [inline doc/howto/autoinstall-quickstart.rst:6]: see above

File: doc/howto/autoinstall-quickstart.rst [hunk 4]
```diff
@@ -1,48 +1,39 @@
...
+For older Ubuntu releases, substitute the version in the name of the ISO image. The instructions should otherwise be the same. See :ref:`<autoinstall-quick-start-s390x>` for instructions on installing on the s390x architecture.
 
 Providing the autoinstall data over the network
-===============================================
+-----------------------------------------------
 
-This method is the one that generalises most easily to doing an entirely
-network-based installation where a machine boots over a network and then is automatically
-installed.
+This method describes a network-based installation. A machine boots over a network and the operating system is automatically installed.
```
Comment (dbungert) [inline doc/howto/autoinstall-quickstart.rst:13]: That's not quite correct.  Netboot is a thing, but what we're describing here isn't netboot.  This text is trying to say that this method of supplying autoinstall is one that we think will also be convenient if someone is doing netboot.

File: doc/howto/autoinstall-quickstart.rst [hunk 5]
```diff
@@ -140,52 +126,54 @@ Create your user-data and meta-data files
...
-``autoinstall`` to the kernel command line, the installer prompts for
-confirmation before touching the disk.
+        -cdrom ~/Downloads/ubuntu-<version-number>-live-server-amd64.iso
 
-The installer reboots at the end but the ``-no-reboot`` flag to ``kvm`` means
-that ``kvm`` will exit when this happens.
+This command boots the system and runs the installation. The installer prompts for a confirmation before modifying the disk. To skip the need for a confirmation, interrupt the booting process, and add the ``autoinstall`` parameter to the kernel command line.
 
-The whole process should take about 5 minutes.
+The installer reboots at the end. The ``-no-reboot`` option to the ``kvm`` command instructs ``kvm`` to exit on reboot. This command takes approximately 5 minutes.
```
Comment (dbungert) [inline doc/howto/autoinstall-quickstart.rst:166]: See above timing comment.

File: doc/howto/autoinstall-quickstart-s390x.rst (hunk 2 above)
Comment (rkratky) [reply doc/howto/autoinstall-quickstart-s390x.rst:115]: Ack

File: doc/howto/autoinstall-quickstart-s390x.rst (hunk 1 above)
Comment (rkratky) [reply doc/howto/autoinstall-quickstart-s390x.rst:6]: Ack

File: doc/howto/autoinstall-quickstart.rst (hunk 4 above)
Comment (rkratky) [reply doc/howto/autoinstall-quickstart.rst:13]: How about:

"This method describes a network-based installation. When booting over a network, use this method to deliver autoinstall to perform the installation."

Comment (dbungert) [reply doc/howto/autoinstall-quickstart.rst:13]: Go for it

File: doc/howto/autoinstall-quickstart.rst [hunk 6]
```diff
@@ -1,48 +1,39 @@
-.. _autoinstall_quickstart:
+.. _autoinstall_quick-start:
```
Comment (Chris-Peterson444) [inline doc/howto/autoinstall-quickstart.rst:1]: See previous comment.

File: doc/howto/autoinstall-quickstart.rst [hunk 7]
```diff
@@ -1,48 +1,39 @@
...
-autoinstall in a VM on your machine.
+This guide provides instructions to use autoinstall with a current version of Ubuntu for the amd64 architecture in a virtual machine (VM) on your computer.
 
-This page assumes that you are installing a recent Ubuntu release. However,
-for older releases, you can substitute the name of the ISO image but the
-instructions should otherwise be the same.
-
-This page also assumes you are on the AMD64 architecture. There is a
-:ref:`version for s390x<autoinstall-quickstart-s390x>` too.
+For older Ubuntu releases, substitute the version in the name of the ISO image. The instructions should otherwise be the same. See :ref:`<autoinstall-quick-start-s390x>` for instructions on installing on the s390x architecture.
```
Comment (Chris-Peterson444) [inline doc/howto/autoinstall-quickstart.rst:8]: Actually, this ref isn't generating a link either. Is this just an side effect of the build site?

File: doc/howto/index.rst [hunk 8]
```diff
@@ -1,29 +1,25 @@
...
+--------------------------------
 
 .. toctree::
    :maxdepth: 1
    
-   Autoinstall quick-start guide <autoinstall-quickstart>
-   Autoinstall quick-start guide for s390x <autoinstall-quickstart-s390x>
-
+   Autoinstall quick start <autoinstall-quickstart>
+   Autoinstall quick start for s390x <autoinstall-quickstart-s390x>
```
Comment (Chris-Peterson444) [inline doc/howto/index.rst:19]: These work, so my above comments may be incorrect.

**Review by dbungert — APPROVED** [review]

File: doc/howto/autoinstall-quickstart-s390x.rst [hunk 9]
```diff
@@ -1,33 +1,31 @@
...
+Autoinstall quick start for s390x
+=================================
 
-This how-to provides basic instructions to perform an automatic installation
-in a virtual machine (VM) on a local machine on the s390x architecture.
+This guide provides instructions to use autoinstall with a current version of Ubuntu for the s390x architecture in a virtual machine (VM) on your computer.
 
-This how-to is a version of :ref:`autoinstall_quickstart`.
-adapted for s390x.
+For older Ubuntu releases, substitute the version in the name of the ISO image. The instructions should otherwise be the same. See :ref:`<autoinstall-quick-start>` for instructions on installing on the amd64 architecture.
```
[the remark this answers is not in the document]
Comment (rkratky) [reply doc/howto/autoinstall-quickstart-s390x.rst:8]: Thanks. It behaves strangely, but I seem to have fixed it now.

**Pull request merged**